custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -92,15 +92,11 @@ def _state_aggregation_query(self, state_id):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
-        migrated = (
-            "agg_migration.is_migrated IS DISTINCT FROM 0"
-            "AND agg_migration.migration_date::date < {start_month_string}"
-        ).format(start_month_string=start_month_string)
-        not_registered = (
-            "agg_availing.is_registered = 0"
-            "AND agg_availing.registration_date::date < {start_month_string}"
-        ).format(start_month_string=start_month_string)
-        seeking_services = "NOT {not_registered} AND NOT {migrated}".format(
+        migrated = "(agg_migration.is_migrated = 1 AND agg_migration.migration_date::date < {start_month_string})::boolean".format(
+            start_month_string=start_month_string)
+        not_registered = "(agg_availing.is_registered = 0 AND agg_availing.registration_date::date < {start_month_string})::boolean".format(
+            start_month_string=start_month_string)
+        seeking_services = "NOT COALESCE({not_registered},'f') AND NOT COALESCE({migrated}, 'f')".format(
```

Comment:
these coalesces are not necessary if you use `DISTINCT FROM` like you had before. why did you remove that?

Reply:
you mentioned of using boolean instead of Integers that's why I am using this